src/renderers/WebGLRenderer.js

```diff
@@ -1339,13 +1339,27 @@ class WebGLRenderer {
 
 			const isWebGL2 = capabilities.isWebGL2;
 
+			const currentRenderTarget = _this.getRenderTarget();
+
+			// If the current render target has a stencil buffer, then the transmission render target also needs it so that it's rendered the same way.
+
+			const needsStencilBuffer = currentRenderTarget !== null ? currentRenderTarget.stencilBuffer : _this.getContextAttributes().stencil;
+
+			if ( _transmissionRenderTarget !== null && needsStencilBuffer === true && _transmissionRenderTarget.stencilBuffer === false ) {
```

Comment:
If we want to save memory at any cost, should this section also test for the opposite case? Meaning the current render target has no stencil buffer but the transmission render target has (because of a previous rendering with stencil).

Reply:
Maximum memory consumption is what matters most. And only using a stencil buffer once and then switching to rendering without a stencil buffer is such a rare corner case that I don't think we need to worry about that.